react/Autosuggest/__snapshots__/Autosuggest.test.js.snap

```diff
@@ -139,7 +137,6 @@ exports[`Autosuggest should render with simple props 1`] = `
       <input
         aria-autocomplete="list"
         aria-expanded="false"
-        aria-haspopup="false"
```

Comment:
These snapshot changes only occurred when I updated to the latest npm flexible dependency versions. They don't appear to be related to the change.